### PR TITLE
Add Ocean batch submit script

### PR DESCRIPTION
**Pull request by geoffrey4444**
## Proposed changes

Adds a standard submission script for running jobs on the Ocean cluster at Cal State Fullerton.

### Types of changes:

- [ ] Bugfix
- [x] New feature
- [ ] Refactor

### Component:

- [ ] Code
- [ ] Documentation
- [x] Build system
- [ ] Continuous integration

### Code review checklist

- [x] The PR passes all checks, including unit tests and `clang-tidy`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [x] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [x] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

**Review by nilsdeppe — DISMISSED** [review]

**Comment by kidder** [comment]
@nilsdeppe told me this needs updating to use singularity...

**Comment by geoffrey4444** [comment]
@kidder I will create a separate submit script for singularity. This PR is for running spectre with a non-singularity build, which I still want to maintain

**Comment by kidder** [comment]
okay, then please have one of your students verify that it works and provide a review for this PR

**Comment by kidder** [comment]
@geoffrey4444 should I close this?

**Comment by geoffrey4444** [comment]
No, please give me a bit more time to update it using singularity

**Comment by geoffrey4444** [comment]
@nilsdeppe I just updated this to use our singularity container, and I tested it on latest develop, that it works as expected on ocean.

File: support/SubmitScripts/Ocean.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+# Flag to stop blas in CCE from parallelizing without charm++
+export OPENBLAS_NUM_THREADS=1
+
+# The 19 is there because Charm++ uses one thread per node for communication
+# Here, -np should take the number of nodes (must be the same as --nodes
+# in the #SBATCH options above).
+SPECTRE_COMMAND="${SPECTRE_EXECUTABLE} +ppn 19 +pemap 0-18 +commap 19"
+
+mpirun -np 2 --map-by ppr:1:node singularity exec \
```
Comment (kidder) [inline support/SubmitScripts/Ocean.sh]: instead of 2 use ${SLURM_JOB_NUM_NODES} and they will always be consistent

**Review by kidder — CHANGES_REQUESTED** [review]

**Comment by nilsdeppe** [comment]
@geoffrey4444 this is waiting on you

**Comment by geoffrey4444** [comment]
Didn't see @kidder's review until just now...I just pushed a fixup

**Comment by nilsdeppe** [comment]
LGTM. Please squash :) I'm sure Larry can handle the small change ;)

**Review by kidder — APPROVED** [review]

**Pull request merged**